src/test/groovy/au/org/ala/profile/hub/AdminControllerSpec.groovy

```diff
@@ -6,15 +6,15 @@ import spock.lang.Specification
 
 class AdminControllerSpec extends Specification implements ControllerUnitTest<AdminController> {
 
-    AdminController mockAdminController
+    AdminController controller
 
     def setup() {
-        mockAdminController = Mock(AdminController)
+        controller = new AdminController()
```

Comment:
controller is already created and is available as a property on this object. no need to create a controller